apis/cache/v1beta1/replication_group_types.go

```diff
@@ -337,7 +337,18 @@ type ReplicationGroupParameters struct {
 	// (http://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/SubnetGroups.html).
 	// +immutable
 	// +optional
-	CacheSubnetGroupName *string `json:"cacheSubnetGroupName,omitempty"`
+	CacheSubnetGroupName string `json:"cacheSubnetGroupName,omitempty"`
```

Comment:
@krishchow I believe this should still be `*string` since it is optional

Reply:
Switched it back to a *string :)